source/concurrent/Promise.ooc

```diff
@@ -109,7 +109,7 @@ Future: abstract class <T> {
 }
 
 _ThreadFuture: class <T> extends Future<T> {
-	_result: Object
+	_result: __onheap__ T
```

Comment:
Thanks, I think we should be less explicit about memory management if we can.
I'd love to see this done automatically, but I think @thomasfanell or @fredrikbryntesson should be the one to decide.